projects/services/consensus-service-typescript/src/consensus-service.ts

```diff
@@ -31,7 +30,7 @@ export default class ConsensusService extends Service {
 
   @Service.SilentRPCMethod
   public async gossipMessageReceived(rpc: types.GossipMessageReceivedContext) {
-    const obj: any = JSON.parse(rpc.req.Buffer.toString("utf8"));
-    this.consensus.gossipMessageReceived(rpc.req.FromAddress, rpc.req.MessageType, obj);
+    const obj: any = JSON.parse(rpc.req.buffer.toString("utf8"));
```

Comment:
I suggest we start using `payload` instead of `obj`.

Reply:
Sure thing